relayer/message_relayer.go

```diff
@@ -483,55 +465,6 @@ func (r *messageRelayer) createSignedMessageAppRequest(requestID uint32) (*avala
 	return nil, errNotEnoughSignatures
 }
 
-func (r *messageRelayer) getCurrentCanonicalValidatorSet() ([]*avalancheWarp.Validator, uint64, error) {
```

Comment:
Signing subnet check moved to `messageRelayer` constructor; the rest of this function moved to `CanonicalValidatorClient.GetCurrentCanonicalValidatorSet`